example/basicauth/main.go

```diff
@@ -6,6 +6,12 @@
 // The basicauth command demonstrates using the github.BasicAuthTransport,
 // including handling two-factor authentication. This won't currently work for
 // accounts that use SMS to receive one-time passwords.
+
```

Comment:
Please change blank line 9 to `//` so that the auto-generated godoc will be bundle with this main.